introduction.md

```diff
@@ -1,2 +1,7 @@
 My name is Marta and I'm just following the course *Mastering Github* from *Code School*.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?